### DISCUSSION TITLE
Organization teams: how to give rights?

### DISCUSSION THREAD

**Discussion by JonathanJumper** [community/community#22533] [New to GitHub]
<p>I haven’t been able to use the teams feature as I would’ve wanted to… pretty standard…</p>
<p>So, I have an organizations, with a few repos on that, I have a team of developers, I wanted to create another team of “viewers” for managers, but, as soon as I invite them to the team, they get instant access to the whole organization, so they can do whatever they want in all repos,</p>
<p>I want that team to only read some of the repos in the org,</p>
<p>How can I do that? this is very frustrating</p>

**Comment by jsoref** [comment, marked as answer]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization#setting-base-permissions">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization#setting-base-permissions" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2119212/181071677-56d02881-8dc5-4065-ba9c-7ea2439dca23.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization#setting-base-permissions" target="_blank" rel="noopener nofollow ugc">Setting base permissions for an organization - GitHub Docs</a></h3>

  <p>You can set base permissions for the repositories that an organization owns.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
